Merdification (#8)
Tristramg/blog: content/merdification.md

+++
category = "transport"
date = 2024-12-28
title = "Merdification des transports en France"
+++

> Le terme [merdification](https://fr.wikipedia.org/wiki/Merdification) vient à l’origine du web. Avec le temps des usages très cool deviennent de plus en plus pénibles à utiliser au point de devenir de la merde.

Reviewer: ```suggestion
Le terme [merdification](https://fr.wikipedia.org/wiki/Merdification) vient à l’origine du web. Avec le temps, des usages très cool deviennent de plus en plus pénibles à utiliser, au point de devenir de la merde.
```

> Je tente ici un parallèle avec ce qui se passe dans les transports en communs, en visant tout particulièrement SNCF.

Reviewer: ```suggestion
Je tente ici un parallèle avec ce qui se passe dans les transports en communs, en visant tout particulièrement la SNCF.
```

Author: y’a des considérations de fond « la SNCF » ou « SNCF », mais je ne retrouve plus mes sources, faudra me croire ;)

> Pour le climat, le train, c’est bien, l’autocar c’est toujours bien, mais un peu moins bien. Le covoiturage ça devient limite, à deux dans une voiture encore moins bien puis y’a l’avion et enfin l’autosolisme.

Reviewer: ```suggestion
Pour le climat, le train c’est bien, l’autocar c’est toujours bien, mais un peu moins bien. Le covoiturage ça devient limite, à deux dans une voiture encore moins bien, puis y’a l’avion et enfin l’[autosolisme](https://fr.wiktionary.org/wiki/autosolisme).
```

> On devrait donc tout faire pour que les transports soient le plus simple à prendre et réduire les moments de tensions. Pourtant on à l’impression que SNCF fait tout pour créer des moments de crispation à tous les instants du voyage.

Reviewer: ```suggestion
On devrait donc tout faire pour que les transports soient le plus simple à prendre et réduire les moments de tensions. Pourtant, on à l’impression que SNCF fait tout pour créer des moments de crispation à tous les instants du voyage.
```

## Des horaires au poil, ça me les hérisse

> En France, les horaires sont souvent optimisés pour avoir un remplisassage optimum. Plus de trains circulent en heure de pointe qu’en heure creuse.

Reviewer: ```suggestion
En France, les horaires sont souvent optimisés pour avoir un remplisassage optimum. Davantage de trains circulent en heure de pointe qu’en heure creuse.
```

> D’ailleurs en heure creuse, s’il y a que 20 passagers, souvent on mettra un autocar à la place du TER.

Reviewer: ```suggestion
D’ailleurs en heure creuse, s’il n’y a que vingt passagers, on mettra souvent un autocar à la place du TER.
```

Sauf que voilà, si moi j’ai envie de voyager en milieu de journée, que j’ai raté ma correspondance, que j’ai un rendez-vous le matin… ET BIEN JE NE PEUX PAS VOYAGER.

Bref, arrêtez de trop optimiser vos horaires.

La règle sur le pouce, c’est un train par heure de 5h du matin à minuit. Ça vaut aussi pour les autocars (oui, même les autocars avec un siège sur 4 qui est occupé — ça reste toujours plus écolo qu’être à 4 dans un blablacar. Oui, même les autocars avec 5 personnes en tout et pour tout — ça reste toujours plus écolo que l’autosolisme).

## Les cartes de réductions

> Faites simple. Une carte = une réduc. Pas de spécificité par région, pas de clause aller-retour… Oui, on veut faire payer les pro plus cher. Mais merde, s’ils prennent le train, ben c’est bien aussi, et tant pis si on a pas trouvé le prix max qu’ils seraient prêts à payer.

Reviewer: ```suggestion
Faites simple. Une carte = une réduc. Pas de spécificité par région, pas de clause aller-retour… Oui, on veut faire payer les pro plus cher. Mais merde, s’ils prennent le train, ben c’est bien aussi, et tant pis si on n’a pas trouvé le prix max qu’ils seraient prêts à payer.
```

{{figure(img="cartes_reduc_sncf.webp", alt="Liste des cartes de reduction en France", caption="Sérieusement ?!")}}

## La réservation obligatoire

> C’est pas grave de faire Paris—Lille assis dans par terre ou debout au bar. Laissons les gens voyager s’ils ont besoin de voyager. Supprimons la réservation obligatoire. Les personnes anticipant leur voyage aurant une place assise garantie, les personnes ayant une urgence pourront voyager.

Reviewer: ```suggestion
C’est pas grave de faire Paris—Lille assis par terre ou debout au bar. Laissons les gens voyager s’ils ont besoin de voyager. Supprimons la réservation obligatoire. Les personnes anticipant leur voyage auront une place assise garantie, les personnes ayant une urgence pourront voyager.
```

> Ne faites pas comme [la région grand est](https://www.ter.sncf.com/grand-est/services-contacts/reservation-a-bord).  

Reviewer: ```suggestion
Ne faites pas comme [la région Grand-Est](https://www.ter.sncf.com/grand-est/services-contacts/reservation-a-bord).  
```

Ne faites pas comme [la région Normandie](https://www.ouest-france.fr/normandie/train-en-normandie-la-reservation-devient-obligatoire-pour-voyager-vers-et-depuis-paris-f82effee-ed40-11ec-b46f-021c1e9ecf74).

Si les trains sont structurellement pleins, alors augmentons la fréquence ! Sur les trajets du quotidien, avoir un train toutes les 30 minutes au lieu de toutes les heures, ça change la vie. On peut enfin se permettre une panne de réveil.

Si les infrastrucures sont saturées, construisons de nouvelles infra ! En plus cela rendra le réseau robuste aux intempéries, aux travaux, aux pannes, aux attaques…  
Peut-être que doubler une ligne de TGV n’est pas nécessaire — alors supprimons la première classe ! Faisons des horaires élargis la nuit, faisons deux trains de nuit par jour !

## Les limites de bagages

> Avant la règle c’était « tout ce qu’on pouvait porter seul » et ne pas dépasser des zones de bagage. Maintenant il y a des règles compliquée selon le type de bagage (instrument de musique, poussette, ski…).

Reviewer: ```suggestion
Avant, la règle c’était « tout ce qu’on pouvait porter seul » et ne pas dépasser des zones de bagage. Maintenant, il y a des règles compliquées selon le type de bagage (instrument de musique, poussette, skis…).
```

> Mais globalement on peut toujours transporter autant de choses (2 grosses valises, une petite). On ne va donc pas résoudre le problème des TGV mal foutus (impossible de mettre une valise au dessus des sièges parce que Duplex, pas assez d’espaces bagages parce qu’il faut remplir au max) car on peut toujours transporter autant des bagages, mais on crée de l’incertitude et de l’arbitraire qui rendent le voyage stressant. On sera plus rassuré avec le coffre d’une voiture sans risquer les 50€ d’amende.

Reviewer: ```suggestion
Mais globalement on peut toujours transporter autant de choses (deux grosses valises, une petite). On ne va donc pas résoudre le problème des TGV mal foutus (impossible de mettre une valise au dessus des sièges parce que Duplex, pas assez d’espaces bagages parce qu’il faut remplir au max) car on peut toujours transporter autant des bagages, mais on crée de l’incertitude et de l’arbitraire qui rendent le voyage stressant. On sera plus rassuré avec le coffre d’une voiture sans risquer les 50€ d’amende.
```

> Et je ne parle même pas du traitement des bagages oubliés, qui représente [la première cause de retard](https://www.ville-rail-transports.com/ferroviaire/exclusif-le-nombre-de-bagages-abandonnes-explose-les-retards-des-tgv-aussi/) des trains. Un bagage égaré doit aller aux objets perdus dans l’espoir de le rendre au propriétaire, pas être traité comme une bombe, ce qu’il n’est sont jamais.

Reviewer: ```suggestion
Et je ne parle même pas du traitement des bagages oubliés, qui représente [la première cause de retard](https://www.ville-rail-transports.com/ferroviaire/exclusif-le-nombre-de-bagages-abandonnes-explose-les-retards-des-tgv-aussi/) des trains. Un bagage égaré doit aller aux objets perdus dans l’espoir de le rendre au propriétaire, pas être traité comme une bombe, ce qu’il n’est jamais.
```

Selon ce [vieil article ce checknews](https://www.liberation.fr/checknews/2018/06/27/combien-de-colis-suspects-s-averent-etre-de-vraies-bombes_1661967/), seulement 0,6% des bagages étaient des objets menançants (dont la moitié en Corse).

> Évidemment, les lois liberticides (dites de sécurité intérieure) en sont grandement responsables, pas que SNCF.

Reviewer: ```suggestion
Évidemment, les lois liberticides (dites de sécurité intérieure) en sont grandement responsables, pas que la SNCF.
```

## Les portiques

> Le but de portique était de réduire la fraude et éviter qu’il y ait des personnes indésirables à bord. C’est absolument prodigieux qu’une idée aussi profondément débile ait pu faire le moindre pas en dehors de la machine à café des encravatés de SNCF.

Reviewer: ```suggestion
Le but des portiques était de réduire la fraude et éviter qu’il y ait des personnes indésirables à bord. C’est absolument prodigieux qu’une idée aussi profondément débile ait pu faire le moindre pas en dehors de la machine à café des encravatés de SNCF.
```

Il faut donc maintenant plus de personnel en gare pour aider les gens à passer les portiques.

> Comme la topologie Française fait qu’il arrive qu’il y ait 1200 personnes qui embarquent d’un coup (PARCE QUE LE QUAI DE DÉPART N’EST INDIQUÉ QUE 20 MINUTES AVANT LE DÉPART 😡, il y a d’ailleurs un [très bon Karambolage](https://www.youtube.com/watch?v=27ZKNMNlm6A) sur cette fantaisie française), il faut ouvrir les portiques du quai d’à côté pour fluidifier un peu.  

Reviewer: ```suggestion
Comme la topologie française fait qu’il arrive qu’il y ait 1200 personnes qui embarquent d’un coup (PARCE QUE LE QUAI DE DÉPART N’EST INDIQUÉ QUE 20 MINUTES AVANT LE DÉPART 😡, il y a d’ailleurs un [très bon Karambolage](https://www.youtube.com/watch?v=27ZKNMNlm6A) sur cette fantaisie française), il faut ouvrir les portiques du quai d’à côté pour fluidifier un peu.  
```

Il faut donc du personnel de sécurité en plus pour que les personnes n’aillent pas sur le mauvais quai.

Il faut aussi saisir son numéro de carte de réduction à l’achat pour diminuer les nouvelles fraudes permises par les portiques (les contrôles à bord diminuant avec ceux-ci, cela devient intéressant de risquer d’acheter un billet avec une réduction injustifiée).  
Même acheter son billet de train à la maison devient merdique à cause des maudits portiques.

> Les gares de villes moyennes ne sont plus utilisable sur toute la longueur des quai pour forcer le contrôle à un point précis, provoquant des queues et la mauvaise humaire.

Reviewer: ```suggestion
Les gares de villes moyennes ne sont plus utilisable sur toute la longueur des quai pour forcer le contrôle à un point précis, provoquant des files d’attente et de la mauvaise humeur.
```

> Et si au lieu de continuer la fuite en avant dans le stupide et pénible, on mettait juste plus de personnel à bord qui peuvent aider, informer (et contrôler) les passagers ?

Reviewer: ```suggestion
Et si, au lieu de continuer la fuite en avant dans le stupide et le pénible, on mettait juste plus de personnel à bord pour aider, informer (et contrôler) les passagers ?
```

## Le zèle

C’est si grave si quelqu’un fraude ? Le rôle premier d’un·e « contrôlleur·se » est la sécurité du train, pas de remettre en question les affirmations des voyageurs.  
Si on rate une correspondance, on veut simplement prendre le train suivant. Si on a échangé son billet, ce n’est pas la mort, l’argent est déjà dans les caisses !  

Les faits divers sont nombreux :
- [les musiciens ne pouvant pas voyager avec leurs instruments](https://www.humanite.fr/culture-et-savoir/musique/sncf-la-bataille-du-rail-des-contrebassistes-pour-monter-a-bord-avec-leur-instrument-801325)
- une amende pour [avoir permis à un enfant d’être avec son père](https://www.linternaute.com/actualite/faits-divers/4709201-270-euros-d-amende-sncf-pour-avoir-accepte-de-changer-de-siege-dans-le-train/)
- une amende car [sa mère avait acheté le billet](https://www.20minutes.fr/societe/4053537-20230918-billet-train-nom-mere-doit-payer-330-euros-amende)

> - les [forums](https://www.reddit.com/r/AskFrance/comments/16ilrte/amendes_injustifi%C3%A9es_%C3%A0_la_sncf_%C3%A7a_vous_est_d%C3%A9j%C3%A0/) sont remplis d’histoire de ce genre

Reviewer: ```suggestion
- les [forums](https://www.reddit.com/r/AskFrance/comments/16ilrte/amendes_injustifi%C3%A9es_%C3%A0_la_sncf_%C3%A7a_vous_est_d%C3%A9j%C3%A0/) sont remplis d’histoire de ce genre.
```

Difficile de savoir quelle est la part [des commissions touchées par les contrôleurs](https://www.bfmtv.com/economie/entreprises/transports/commissionnement-des-controleurs-la-sncf-assume-mais-evoque-une-prime-marginale_AV-202403130381.html) et la [politique officielle contre la fraude](https://www.groupe-sncf.com/fr/groupe/coulisses/circulation-trains/fraude-usurpation-identite) du groupe, mais la tolérance et le bénéfice du doute devraient être la norme.

## Des fois ça s’améliore

> De manière inattendu, parfois il y a tout de même grands sauts en avant de simplification, comme la [tarification en Île-de-France](https://www.iledefrance-mobilites.fr/actualites/simplification-tarifs-transports-ile-de-france) que j’applaudis très fort.

Reviewer: ```suggestion
De manière inattendu, il y a tout de même parfois de grands sauts en avant dans la simplification, comme la [tarification en Île-de-France](https://www.iledefrance-mobilites.fr/actualites/simplification-tarifs-transports-ile-de-france) que j’applaudis très fort.
```

> Espérons maintenant que SNCF et les régions organisatrices des mobilités en prendront de la graine et que l’[autorité de régulation des transports](https://www.autorite-transports.fr/) mette son nez sur toutes ces questions.

Reviewer: ```suggestion
Espérons maintenant que SNCF et les régions organisatrices des mobilités en prendront de la graine et que l’[autorité de régulation des transports](https://www.autorite-transports.fr/) mettra son nez dans toutes ces questions.
```
